server/auth/sso/sso.go

```diff
@@ -298,6 +305,11 @@ func (s *sso) HandleCallback(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	value := Prefix + raw
+	if s.compressBearerToken {
+		compressedToken := Prefix + uuid.New().String()
+		s.bearerTokenMap.Add(compressedToken, value)
+		value = compressedToken
+	}
```

Comment:
I was a little confused by the name `compressedToken` and `compressBearerToken` since in my mind that's an actual compression algorithm but I'm okay if others are good with it. Otherwise, I would think something like `tokenRefererence` or maybe `reducedToken`.